base/uk.ac.stfc.isis.ibex.configserver/src/uk/ac/stfc/isis/ibex/configserver/json/JsonConverters.java

```diff
@@ -56,6 +56,13 @@
 		return new JsonDeserialisingConverter<>(Configuration.class, gson).apply(withFunction(INIT_CONFIG));
 	}
 
+    @Override
+    public Converter<String, Collection<Configuration>> toConfigList() {
+        Gson gson = new GsonBuilder().registerTypeAdapterFactory(new LowercaseEnumTypeAdapterFactory()).create();
+        return new JsonDeserialisingConverter<>(Configuration[].class, gson)
+                .apply(Convert.<Configuration>toCollection());
```

Comment:
If you add  `.apply(forEach(INIT_CONFIG))` as in `toComponets` below then each config will get initialised.